clang/test/SemaSYCL/esimd-private-global.cpp

```diff
@@ -0,0 +1,18 @@
+// RUN: %clang_cc1 -fsycl -fsycl-is-device -fsycl-explicit-simd -fsyntax-only -verify -pedantic %s
+
+// no error expected
+__attribute__((opencl_private)) __attribute__((register_num(17))) int privGlob;
+
+// expected-error@+1{{'register_num' attribute takes one argument}}
+__attribute__((opencl_private)) __attribute__((register_num())) int privGlob1;
+
+// expected-error@+1{{'register_num' attribute takes one argument}}
+__attribute__((opencl_private)) __attribute__((register_num(10, 11))) int privGlob2;
+
+// expected-error@+1{{(SYCL explicit SIMD) private global variable cannot have an initializer}}
```

Comment:
Why is SYCL explicit SIMD in parens in this message?  It likely should be just SYCL Explicit SIMD private global....

Reply:
This designates the special mode. I expect there will be other messages tagged similarly, where braces won't make sense.

Reply:
Sorry, I don't get it?  We don't have anything like that elsewhere in the compiler, so this would be a completely new designation.

Reply:
My point is that we'll likely have more of these. This is not adjective to "private global" but the "explicit SIMD" mode tag.

Reply:
SYCL explicit SIMD does not permit private global variable to have an initializer

Reply:
We discussed a reword of this error message on the call, it seems it hasn't happened yet.

Reply:
Right, overlooked this one